components/ws-manager-bridge/src/bridge.ts

```diff
@@ -94,13 +96,43 @@ export class WorkspaceManagerBridge implements Disposable {
             this.config.controllerMaxDisconnectSeconds,
         );
 
+        const tim = setInterval(() => {
```

Comment:
I believe this does not immediately execute `updateWorkspaceClasses` but only after the interval has passed. Is that intended?

Reply:
Yes, I understood that eventually when reading through the diff. My future me would probably search a bit longer to understand that.